src/community/oseo/oseo-rest/src/main/java/org/geoserver/opensearch/rest/AbstractOpenSearchController.java

```diff
@@ -217,7 +223,35 @@ protected OgcLinks buildOgcLinksFromFeature(Feature feature, boolean notFoundOnE
         List<OgcLink> links = Collections.emptyList();
         Collection<Property> linkProperties =
                 feature.getProperties(OpenSearchAccess.OGC_LINKS_PROPERTY_NAME);
-        if (linkProperties != null) {
+
+        Map<String, String> unknownFields = new HashMap<>();
+        // taken from OgcLink properties. these properties are known(static) from OgcLink.
+        Set<String> knownFields =
```

Comment:
This is a constant, please don't declare it in methods, but at the top of the file as a proper constant (static final, all uppercase name).

Reply:
done